### PR TITLE
Adds api docs for one shot object detector

### PR DESCRIPTION
**Pull request by shreyajain17**

### CODE REVIEW

File: src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  create
+  OneShotObjectDetector
+
+utilities
+---------
+.. autosummary::
+  :toctree: generated/
+  :nosignatures:
+
+  util.preview_augmented_images
```
Comment (TobyRoseman) [inline src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst]: Do we also want to add `util.check_one_shot_input`?

Comment (shreyajain17) [reply src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst]: We should ask @shantanuchhabra. It had nothing in comments so I didn't add it yet.

Comment (srikris) [reply src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst]: I don't think we should add that. That;s probably an underscore error.

**Comment by srikris** [comment]
Fixes #1985

File: src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst (hunk 1 above)
Comment (srikris) [reply src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst]: @TobyRoseman The _ error will be fixed in #1986

**Comment by shreyajain17** [comment]
@srikris Sorry I missed that. Will do it!

**Comment by shreyajain17** [comment]
> We should also add the appropriate API docs for the predict, evaluate, and export to CoreML methods.

API Docs for predict, evaluate and export_coreml are in my PR #1997

**Pull request merged**